docker/Dockerfile

```diff
@@ -88,6 +88,13 @@ RUN apt-get update && \
         zip && \
     rm -rf /var/lib/apt/lists/*
 
+# Install Arch dependencies
```

Comment:
Just need to change the container baseline from `bullseye` to `bookworm` - since `pacman-package-manager` is not available in bullseye.

If it works, we can ask the author to backport the package to bullseye-backports and install from there, or just stick with bumping the container baseline. If you can do that in a commit on top, we can always decide that in another PR.

Reply:
Bumped - that does cause debian and aperitis tests to fail. Let me know what you'd prefer - omit the bump, fix those tests, other. 

Once the Arch/pacstrap/pacman CI finishes that it - it'll take some time.

Reply:
Seems like the bump to bookworm is breaking DNS/name resolution for all distros - Debian, Aperitis and Arch.

Reply:
sorry I forgot to mention - in that update you will also need to install the "systemd-resolved" package

Reply:
Fixed thanks.